css/style.css

```diff
@@ -0,0 +1,663 @@
+}
+
+.img-text {
+  padding-right: 20px;
+}
+
+.img-text div {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.telescope-image-container img {
+  height: auto;
+  width: 800px;
+}
+
+.footer-tittle img {
+  margin-right: 60px;
+}
+
+.img-text div img {
+  width: 70px;
+  height: 70px;
+}
+
+.column-grid li p a {
+  font-size: 14px;
+  text-decoration: none;
+  color: rgb(0, 0, 0);
+}
+
+.ul-text-1 li p a {
+  font-size: 12px;
+  text-size-adjust: 100%;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.ul-text-2 li p a,
+.ul-text-3 li p a {
+  font-size: 11.5px;
+  text-size-adjust: 100%;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+}
+
+.main-section-footer-text p a:hover {
+  text-decoration: none;
+}
+
+.img-text div a:hover,
+.corporight p a:hover,
+.column-grid li p a:hover,
+.ul-text-1 li p a:hover,
+.ul-text-2 li p a:hover,
+.ul-text-3 li p a:hover {
+  text-decoration: underline;
+}
+
+.text-image-container {
+  height: 100%;
+  width: 950px;
+}
+
+.text-image-tittle h3 {
+  font-size: 1.2rem;
+  line-height: 10px;
+  font-weight: 600;
+  font-style: normal;
+  margin-top: 18px;
+  margin-bottom: 2px;
+}
+
+.text-image-description {
+  color: #666;
+  font-size: 0.8375rem;
+  line-height: 1.25rem;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  text-align: left;
+}
+
+.grid-container {
+  background-image: url(../images/inflation-945.png);
+  background-size: 950px auto;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  width: 950px;
+  height: auto;
+  grid-column-gap: 22px;
+}
+
+.grid-first-row,
+.grid-second-row {
+  margin-top: 140px;
+  margin-left: 10px;
+  width: 210px;
+  font-size: 13px;
+  line-height: 15px;
+  font-weight: 300;
+  color: #000;
+  font-family: nyt-franklin, arial, helvetica, sans-serif;
+}
+
+.grid-line-border-bottom {
+  border: 0.5px solid rgb(37, 36, 36);
+  width: 72px;
+  margin: 18px auto 0;
+  float: left;
+}
+
+.author {
+  margin-top: 40px;
+}
+
+.video-container {
+  margin-top: 40px;
+  text-align: left;
+  margin-bottom: 40px;
+}
+
+.video-container div {
+  max-width: 85%;
+}
+
+.video-container div p {
+  color: #666;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  text-align: left;
+}
+
+.telescope-image-container {
+  height: 100%;
+  width: 800px;
+  margin-bottom: 30px;
+  margin-top: 30px;
+}
+
+.telescope-image-container div {
+  max-width: 90%;
+}
+
+.telescope-image-container div p {
+  color: #666;
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  text-align: left;
+}
+
+.main-section-footer-text {
+  margin-bottom: 30px;
+}
+
+.comment-button {
+  cursor: pointer;
+  background-color: rgb(86, 123, 149);
+  color: white;
+  border-radius: 3px;
+  text-align: center;
+  border: 0;
+  font-size: 13px;
+  padding: 12px 100px;
+}
+
+.icons-bttns {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
Same style definition of the selector on line 223, group selectors with the same style definition, or create a new class.